pkg/auth/authproviders/util.go

```diff
@@ -14,6 +14,8 @@ const (
 	EmailAttribute  = "email"
 	UseridAttribute = "userid"
 	NameAttribute   = "name"
+	RolesAttribute  = "roles"
+	OrgIDAttribute  = "org_id"
```

Comment:
Any reason to use `org_id` instead of `orgid`? For users, we also use `userid`, I'd prefer if it was consistent.